examples/one_degree_simulation.jl

```diff
@@ -15,92 +16,78 @@ using Dates
+start_date = DateTime(1993, 1, 1) 
+stop_date = DateTime(1993, 12, 1) 
+dates = range(start_date, step=Month(1), stop=stop_date)
+ecco_temperature = Metadata(:temperature; dates, dataset=ECCO4Monthly())
+ecco_salinity = Metadata(:salinity; dates, dataset=ECCO4Monthly())
 
-download_dataset(temperature)
-download_dataset(salinity)
+download_dataset(ecco_temperature)
+download_dataset(ecco_salinity)
 
 # ### Grid and Bathymetry
 
-Nx = 360
-Ny = 180
-Nz = 100
-
-r_faces = exponential_z_faces(; Nz, depth=5000, h=34)
-z_faces = Oceananigans.MutableVerticalDiscretization(r_faces)
+arch = GPU()
+Nx = 180
+Ny = 85
+Nz = 40
 
-underlying_grid = TripolarGrid(arch;
-                               size = (Nx, Ny, Nz),
-                               z = z_faces,
-                               halo = (5, 5, 4),
-                               first_pole_longitude = 70,
-                               north_poles_latitude = 55)
+z = exponential_z_faces(; Nz, depth=4000, h=34)
+underlying_grid = TripolarGrid(arch; size = (Nx, Ny, Nz), halo = (5, 5, 4), z)
+# underlying_grid = LatitudeLongitudeGrid(arch; size = (Nx, Ny, Nz), halo = (5, 5, 4), z, longitude=(0, 360), latitude=(-80, 80))
 
+## 75 interpolation passes smooth the bathymetry near Florida so that the Gulf Stream is able to flow:
 bottom_height = regrid_bathymetry(underlying_grid;
-                                  minimum_depth = 10,
-                                  interpolation_passes = 75, # 75 interpolation passes smooth the bathymetry near Florida so that the Gulf Stream is able to flow
-				                  major_basins = 2)
+                                  minimum_depth = 20,
+                                  interpolation_passes = 100,
+                                  major_basins = 2)
 
 # For this bathymetry at this horizontal resolution we need to manually open the Gibraltar strait.
-# view(bottom_height, 102:103, 124, 1) .= -400
 grid = ImmersedBoundaryGrid(underlying_grid, GridFittedBottom(bottom_height); active_cells_map=true)
 
 # ### Restoring
 #
 # We include temperature and salinity surface restoring to ECCO data.
 restoring_rate  = 1 / 10days
-z_below_surface = r_faces[end-1]
-
-mask = LinearlyTaperedPolarMask(southern=(-80, -70), northern=(70, 90), z=(z_below_surface, 0))
-
-FT = ECCORestoring(temperature, grid; mask, rate=restoring_rate)
-FS = ECCORestoring(salinity,    grid; mask, rate=restoring_rate)
-forcing = (T=FT, S=FS)
+mask = LinearlyTaperedPolarMask(southern=(-80, -70), northern=(70, 90), z=(z[1], 0))
+@inline damping(x, y, z, t, u, p) = - p.r * p.mask(y, z) * u
+FT = ECCORestoring(ecco_temperature, arch; mask, rate=restoring_rate)
+FS = ECCORestoring(ecco_salinity, arch; mask, rate=restoring_rate)
+Fu = Forcing(damping, field_dependencies=:u, parameters=(r=restoring_rate; mask))
+Fv = Forcing(damping, field_dependencies=:v, parameters=(r=restoring_rate; mask))
+forcing = (T=FT, S=FS, u=Fu, v=Fv)
```

Comment:
this looks right to me, but the docs built is complaining about something in continuous_forcing: https://buildkite.com/clima/climaocean-examples/builds/2996#0196470a-4e03-40b2-84c7-56de1e947d32/22-792